srml/grandpa/src/lib.rs

```diff
@@ -128,11 +128,6 @@ pub trait Trait: system::Trait {
 	type Log: From<Log<Self>> + Into<system::DigestItemOf<Self>>;
 
 	/// The session key type used by authorities.
-	#[cfg(not(feature = "std"))]
-	type SessionKey: Parameter + Default + MaybeSerializeDebugButNotDeserialize;
-
-	/// The session key type used by authorities.
-	#[cfg(feature = "std")]
```

Comment:
are these changes related to to this PR? they appear to be causing tests to fail